tests/es/climate_HassClimateSetTemperature.yaml

```diff
@@ -2,22 +2,30 @@ language: "es"
 tests:
   - sentences:
       - "pon la temperatura a 30"
+      - "sube la temperatura a 30"
+      - "baja la temperatura a 30"
```

Comment:
have you considered "configura la temperatura a 30"? I can see myself saying something like that when speaking to my oven or my thermostat. But admittedly I'm a bit of a snob.

Reply:
Sure, you can add one extra commit to your branch and the PR will reflect that. Although if you made this using github's web user interface, I'm not sure how to do it. 

In any case, it's not important. I'd cool with merging this as is and keep making improvements later.

Reply:
Found the way. Changes committed. Thanks for your input.